src/runtime/test/render-vdom.spec.tsx

```diff
@@ -1097,5 +1097,93 @@ describe('render-vdom', () => {
       await waitForChanges();
       expect(rootInstance.counter).toEqual(2);
     });
+
+    it('should not call ref cb w/ null when children are reordered', async () => {
+      @Component({ tag: 'cmp-a' })
+      class CmpA {
+        counter = 0;
```

Comment:
Does this field ever get used? Can we remove it?

Reply:
ahh indeed it does not 🤦‍♀️